### DISCUSSION TITLE
Executing multiple queries

### DISCUSSION THREAD

**Discussion by cloud-rocket** [mkleehammer/pyodbc#1348] [Q&A]
### Environment

- Python: 3.11 64 bit
- pyodbc: 
- OS: Windows
- DB: Aspen IP21
- driver: AspenTech SQLplus

### Issue

Because Aspen IP21 is not fully compliant with MSSQL, there is no way to limit the amount of results without running 2 queries together:
```
SET MAX_ROWS 100;
SELECT SELECT NAME, TS, VALUE from HISTORY where NAME='abc';
```

Seems like `cursor.execute` only runs the 1st query, but not the second.

expected behavior: Return 100 rows as it happens in Aspen SQLPlus
observed behavior: Error returned

**Comment by v-chojas** [comment]
Did you try `cursor.nextset()` ?
